Index: src/Main.java
===================================================================
diff --git a/src/Main.java b/src/Main.java
deleted file mode 100644
--- a/src/Main.java	(revision 053ad8af0dbb2fc5a03e101eff17bf155b002af9)
+++ /dev/null	(revision 053ad8af0dbb2fc5a03e101eff17bf155b002af9)
@@ -1,5 +0,0 @@
-public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
-    }
-}
\ No newline at end of file
